### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent incorrect error when multi-line function call starts on same line as open tag

### ISSUE THREAD

**Issue by buttflattery** [squizlabs/PHP_CodeSniffer#2337]
I am getting the error 

> Line indented incorrectly; expected 0 spaces, found 16

for the following code which has the HTML and PHP mixed code and the indentation is correct, but it gives me this error on the `?>`  which looks like it expects it to be at the start of the line rather than being indented under the opening tag `<?php` position, any ideas ?

```
<?php

    use yii\helpers\Html;
    use yii\widgets\DetailView;

    /* @var $this yii\web\View */
    /* @var $model common\models\Message */

    $this->title = $model->name;
    $this->params['breadcrumbs'][] = ['label' => 'My Plan'];
    $this->params['breadcrumbs'][] = $this->title;
?>
<div class="block-header">
    <div class="card">
        <div class="header">
            <h2>
                <?php echo Html::encode($this->title) ?>
            </h2>

        </div>
        <div class="body">
            <div class="body table-responsive">
                <?php echo
                    DetailView::widget(
                        [
                            'model' => $model,
                            'options' => ['class' => 'table'],
                            'attributes' => [
                                'description:ntext',
                                [
                                    'attribute' => 'price',
                                    'value' => Yii::$app->formatter->format($model->price, 'currency'),
                                ],
                                'max_subscribers',
                                'max_responders',
                                'max_campaigns',
                                'max_dedicated_numbers',
                            ],
                        ]
                    );
                ?>
            </div>
        </div>
    </div>
</div>

```

**Comment by gsherwood** [comment]
I'm pretty sure this is an issue with starting code on the same line as the PHP opening tag. Can be reproduced with this code:
```php
<div class="block-header">
    <?php echo
        foo(
            $foo
        );
    ?>
</div>
```

**Comment by buttflattery** [comment]
@gsherwood i dont think that starting the code on the same line as the `<?php` starting tags are on causes this, see the image below, it still continues to highlight the closing `?>` php tag as in correctly indented. saying 

> Line indented incorrectly; expected 8 spaces, found 4

![image](https://user-images.githubusercontent.com/1536454/51108944-823e1f80-1815-11e9-8d6f-8e6ff10bda9c.png)

**Comment by gsherwood** [comment]
Code to reproduce the second issue:
```php
<div>
    <?php
        echo foo(
        );
    ?>
</div>
```

**Comment by gsherwood** [comment]
The second issue is not actually a bug. The sniff assumes the closing parenthesis is setting where the indent level of the code (as a closing curly brace would) and so uses that to expect an 8-space indent.

The sniff doesn't like it when top-level code is indented from the opening tag. See this comment: https://github.com/squizlabs/PHP_CodeSniffer/issues/2356#issuecomment-453869114

The first issue is a real issue and is being fixed.

**Comment by gsherwood** [comment]
I've pushed a fix for the original issue, which will be in 3.4.1. Thanks for the report.
